initial (#925)
ealush/vest: website/docs/writing_your_suite/result_object.md

[Read more about groups](../writing_tests/advanced_test_features/grouping_tests.md)

## `getError` and `getWarning`

These functions return a string error for the specified field.

```js
resultObject.getError('username');
// 'Username is too short'

resultObject.getWarning('password');
// 'Password must contain special characters'
```

> If there is no error for the field, the function defaults to an empty string.
> 
> ```js
> resultObject.getError('username');
> // ""
> 
> resultObject.getWarning('username');
> // ""
> ```

Reviewer: ```suggestion
If there is no error for the field, the function will return undefined.

```js
resultObject.getError('username');
// undefined

resultObject.getWarning('username');
// undefined
```
```
